system/security/LdapSecurity/permissions.cpp

```diff
@@ -77,10 +77,10 @@ unsigned PermissionProcessor::ldap2sec(unsigned ldapperm)
     if(ldapperm & 0x28)
         permission |= SecAccess_Write;
```

Comment:
I agree that the bitfield handling of permissions is sloppy if not straight up wrong, in that setting SecAccess_Write (7)  grants SecAccess_Access (1) and SecAccess_Read (3). But that's another Jira....